app/templates/components/settings/contact-info-section.hbs

```diff
@@ -1,11 +1,11 @@
 <form class="ui form" {{action 'submit' on = 'submit'}}>
   <div class="field">
     <label>Email</label>
-    <input type="text" name="email" placeholder="Email">
+    {{input type="text" name="email" placeholder="Email"}}
```

Comment:
And don't keep the placeholder text the same as the label text. it's already there in one place. Either completely remove placeholder or keep a sample text such as `name@example.com` ... I'd suggest completely remove placeholder.